docs/main/faqs/faqs.md

```diff
@@ -670,6 +670,14 @@ The `mayastor/mayastor` Helm chart, along with other stable OpenEBS storage plug
 
 [Go to top](#top)
 
+### Why does a snapshot operation fail after a node reboot?
```

Comment:
The question is not entirely true, it fails after reboot only if the pod is not restarted or staged after reboot.

Reply:
@tiagolobocastro @avishnu @dsharma-dc Any better way to frame this?

Reply:
fail sometimes? or
What can cause a snapshot operation to sometimes fail after a node reboot?